src/Frontend/Path.hs

```diff
@@ -106,7 +106,7 @@ import Types
 
 -- * types
 
--- FIXME: Introduce AllowGet only
+-- FIXME: Introduce AllowGet only, or better make it a list of [GET|POST].
```

Comment:
interesting questions.  but i think you can implement set membership tests in typed lists.  details to be discussed once we get around to playing with this...